packages/core/src/Stores/contract-store.js

```diff
@@ -211,15 +210,6 @@ export default class ContractStore extends BaseStore {
                     },
                     is_dark_mode
                 );
-                this.marker = this.root_store.client.is_beta_chart
-                    ? calculateMarker(this.contract_info, this.root_store.ui.is_dark_mode_on, false, {
-                          accu_high_barrier: this.accu_high_barrier,
-                          accu_low_barrier: this.accu_low_barrier,
-                      })
-                    : calculate_marker(this.contract_info, {
```

Comment:
We don't need this `calculate_marker`(...) at all? Asking because in other parts you left parts of ternary

Reply:
We are using `calculateMarker` function from helper that was created for markers in flutter charts. For this version , we don't need calculate_marker .

Reply:
Oh, I see now, thanks for reply)